app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide(:title, "Home") %> <!DOCTYPE html>
+
+
+<h1>Bamgram</h1>
+<p>To get started, <a href="users/new">sign-up</a> or log-in</p>
```

Comment:
Minor detail - can you replace this with rails helper method `link_to`, as in something like 
`<p>To get started <%= link_to "Sign up", new_user_path %>or <%=link_to "Log in", login_path %></p>